configure.py

```diff
@@ -398,17 +398,6 @@
     dest='with_etw',
     help='build with ETW (default is true on Windows)')
 
-parser.add_option('--use-largepages',
```

Comment:
Were these build configuration options experimental? I can’t see any indication that they were. If they weren’t should we be removing options from `configure` during LTS (as opposed to, say, making them no-ops if they no longer do anything)?

Cc @nodejs/lts

Reply:
I’d be a fan of making them no-ops instead of removing them regardless of experimental status and/or LTS release line targeting.

Reply:
@addaleax I'll file another PR against master re-introducing the option as a no-op and then I'll include the commit into the backport.

Reply:
No-ops might not be the best choice since this previously would actually *enable* the largepages during runtime if this option was provided during build time. Which is currently the equivalent of calling node with `--use-largepages=silent`. Therefore those that used a custom build will have this silently disabled.
Should we perhaps make it so that largepages are enabled if build-time option was provided? Or maybe at least print a warning during build that runtime option must be used instead?

Reply:
@lundibundi PTAL at #31103!